src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/dialog/preferences/PreferencesPages.kt

```diff
@@ -1010,6 +1011,65 @@ object PreferencesPages {
             }
     }
 
+    object EditorBorderHighlight : PreferencesPage(
+        Strings.PreferencesEditorBorderHighlight,
+        Strings.PreferencesEditorBorderHighlightDescription,
+    ) {
+        override val content: List<PreferencesGroup>
+            get() = buildPageContent {
+                withContext(
+                    selector = { it.editor },
+                    updater = { copy(editor = it) },
+                ) {
+
+                    switch(
```

Comment:
Let's use grouping (you can find examples in this file) to make it more clear, and some repeated texts can be simplified.

![QQ_1730900533308](https://github.com/user-attachments/assets/020d5e13-398f-4f89-bf78-277330bb88f9)

For example, these can be simply `Color` and `Width` under a group.

Reply:
Also I feel like the default color/width settings should be inverted.
Cursor should have the highest visual priority.